common/src/main/java/com/viaversion/viabackwards/protocol/protocol1_12_2to1_13/packets/BlockItemPackets1_13.java

```diff
@@ -189,6 +203,13 @@ public void register() {
                     int blockState = wrapper.read(Type.VAR_INT);
                     Position position = wrapper.get(Type.POSITION1_8, 0);
 
+                    // Note block special treatment
+                    if (blockState >= 249 && blockState <= 748) { // Note block states id range
+                        wrapper.user().get(NoteBlockStorage.class).storeNoteBlockUpdate(position, blockState);
+                        wrapper.cancel();
+                        return;
```

Comment:
The storage checkAndStore still needs to be called and you can't just cancel the block change from air

Reply:
Fixed